extras/indexing/src/test/java/org/apache/rya/indexing/mongo/MongoEntityIndexTest.java

```diff
@@ -54,11 +55,11 @@
                     .build());
     private static final RyaURI RYA_PERSON_TYPE = new RyaURI("urn:person");
 
-    private static EntityIndexOptimizer optimizer;
-    private static EntityStorage entityStorage;
+    private EntityIndexOptimizer optimizer;
+    private EntityStorage entityStorage;
 
-    @BeforeClass
-    public static void beforeClass() throws Exception {
```

Comment:
Decided to change this to @Before as opposed to changing parent init to @BeforeClass?